app/components/primer/navigation/tab_component.rb

```diff
@@ -106,6 +99,7 @@ def initialize(list: false, selected: false, with_panel: false, icon_classes: ""
 
         @wrapper_arguments = wrapper_arguments
         @wrapper_arguments[:tag] = :li
+        @wrapper_arguments[:role] = :presentation if with_panel
```

Comment:
[Read more about presentation role](https://www.w3.org/TR/wai-aria-practices/#presentation_role)

Reply:
This issue comment might make a good code comment ❤️